src/index.js

```diff
@@ -1,29 +1,56 @@
 import './style.css';
-import { add, display } from './crud.js';
+import { createGame, sendData, getData } from './crud.js';
 
 const name = document.getElementById('name');
 const score = document.getElementById('score');
 
 const subBtn = document.querySelector('.subBtn');
 const refreshBtn = document.querySelector('.refreshBtn');
+const scoreSec = document.querySelector('.listScores');
 
-// setting up array of scores
-let scores = [];
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
 
-// setting up localstorage
-if (JSON.parse(localStorage.getItem('scores') !== null)) {
-  scores = JSON.parse(localStorage.getItem('scores'));
-} else {
-  localStorage.setItem('scores', JSON.stringify(scores));
-}
+const getIdFromStorage = () => {
+  const localStorageId = localStorage.getItem('id') ? JSON.parse(localStorage.getItem('id')) : null;
+  return localStorageId;
+};
+
+const display = async () => {
+  scoreSec.innerHTML = '';
+  const { result } = await getData(`${baseURL}games/${getIdFromStorage()}/scores`);
+  result.forEach((itemScore) => {
+    const eachScore = `<p class="eachScore">${itemScore.user}:${itemScore.score}</p><hr>`;
+    scoreSec.insertAdjacentHTML('beforeend', eachScore);
+  });
+};
```

Comment:
- [ OPTIONAL] We don't need to create a new game every time the user opens a new page from the browser, it would be better to get the data from the created game's endpoint instead of creating new a new one.